.github/workflows/ci.yaml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - main
+    paths-ignore:
+    - '**.md'
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    env:
+      FORCE_COLOR: true
+    steps:
+    - uses: actions/checkout@v2
+    # Based on https://github.com/snyk/snyk/blob/master/.github/workflows/smoke-tests.yml
+    
+    - name: Install ShellSpec
+      run: |
+        curl -fsSL https://git.io/shellspec | sh -s -- -y
```

Comment:
let's not download & run scripts like this. Please download and save shellspec in `bin/` folder on the repo (can you please do the same for https://github.com/faros-ai/faros-events-cli ? )

Reply:
Using the same approach as @willmarks in https://github.com/faros-ai/faros-events-cli/pull/85

Thank you @willmarks !!!